lib/board.rb

```diff
@@ -0,0 +1,61 @@
+class Board
+  attr_accessor :cells, :position
+
+  @cells = []
+  def initialize
+    @cells = Array.new(9, " ")
+  end
+
+  def reset!
+    self.cells.clear
```

Comment:
No need to call `clear` as the contents of `cells` will be overwritten by the assignment.